x-aeson/test/Test/X/Data/Aeson.hs

```diff
@@ -1,13 +1,18 @@
+{-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell #-}
 module Test.X.Data.Aeson where
 
-import           Control.Applicative
-
+import           Data.Either
+import           Data.Functor ((<$>))
+import           Data.Maybe
 import           Data.Text as T
 
 import           Disorder.Aeson
 
+import           GHC.Base
+import           GHC.Show
```

Comment:
Not sure what these imports are for but you can get the `Show` type class in `Text.Show`